libvast/src/system/type_registry.cpp

```diff
@@ -201,11 +206,16 @@ type_registry(type_registry_actor::stateful_pointer<type_registry_state> self,
     },
     [self](atom::load) -> caf::result<atom::ok> {
       VAST_DEBUG("{} loads taxonomies", self);
+      std::error_code err{};
       auto dirs = get_schema_dirs(self->system().config());
       concepts_map concepts;
       models_map models;
       for (const auto& dir : dirs) {
-        if (!exists(dir))
+        auto dir_exists = std::filesystem::exists(dir, err);
+        if (err)
```

Comment:
I think we should do `err.clear()`. In the case where more than one directory doesn't exist (and since we continue in the loop), I think we want to warn on just the ones that don't exist rather than erroneously warn on all of the dirs remaining after the first one fails (and thus `err` is set). Does that make sense? 🤔

Reply:
`exists()` calls `err.clear()` itself if it didn't fail. See https://en.cppreference.com/w/cpp/filesystem/exists.

Reply:
Ah, perfect then. 👍